README.md

```diff
@@ -124,16 +124,19 @@ Animated GIFs will be displayed as still images to reduce loading time and the
 impact on performance of a client website.
 
 
-dev installation
+Dev installation
 ----------------
 
 ```
 git clone git@github.com:styladev/widget.git
 cd widget
-npm i
+npm install
 npm run build
+python -m SimpleHTTPServer
```

Comment:
actually instead of a shortcut I'd prefer a docker setup that takes care of it, but I didn't want to delve into that right now. And I added the python server just because I didn't know a better javascript alternative at that time, so if you have a simpler setup, we can use it